### PR TITLE
chore: remove Bobgy from OWNERS

### PR DESCRIPTION
**Pull request by Bobgy**

### CODE REVIEW

File: content/en/docs/components/multi-tenancy/OWNERS [hunk 1]
```diff
@@ -1,4 +1,2 @@
-approvers:
-  - Bobgy
```
Comment (zijianjoy) [inline content/en/docs/components/multi-tenancy/OWNERS:2]: Can I keep you here until we find a replacement?

File: content/en/docs/distributions/OWNERS [hunk 2]
```diff
@@ -1,6 +1,5 @@
 approvers:
-  - Bobgy
```
Comment (zijianjoy) [inline content/en/docs/distributions/OWNERS:2]: Can you add me here?

**Comment by zijianjoy** [comment]
@chensun for review as well.

**Comment by Bobgy** [comment]
@zijianjoy you are the root owner. Feel free to update after this is merged.

**Comment by chensun** [comment]
/lgtm
/approve 

Thanks, @Bobgy !

**Pull request merged**